code/lang/ILanguage.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+using Sandbox;
+
+namespace TTTReborn.Language
+{
+    public interface ILanguage
```

Comment:
Pretty uncommon to use Interfaces like this (as classes), I don't see you adding them to any class. I think, we would go better with a (partial?) class here

Reply:
Agreed, an interface shouldn't be used here. Could just go with a static class.